index.html

```diff
@@ -248,31 +248,56 @@ <h3>About Myself</h3>
+                      <path fill-rule="evenodd" clip-rule="evenodd" d="M5.29289 8.29289C5.68342 7.90237 6.31658 7.90237 6.70711 8.29289L12 13.5858L17.2929 8.29289C17.6834 7.90237 18.3166 7.90237 18.7071 8.29289C19.0976 8.68342 19.0976 9.31658 18.7071 9.70711L12.7071 15.7071C12.3166 16.0976 11.6834 16.0976 11.2929 15.7071L5.29289 9.70711C4.90237 9.31658 4.90237 8.68342 5.29289 8.29289Z" fill="#5E6C84"/>
+                    </svg></li></ul>
+                    <ul>
+                      <li><img src="asset/images/javascriptlogo.png" alt="lang-logo"><span>Javascript</span></li>
+                      <li><img src="asset/images/htmllogo.png" alt="lang-logo"><span>HTML</span></li>
+                      <li><img src="asset/images/csslogo.png" alt="lang-logo"><span>CSS</span></li>
+                  </ul>
+                  <ul class="work"><li><b>Frameworks</b></li>
+                      <li><svg xmlns="http://www.w3.org/2000/svg" width="24" height="25" viewBox="0 0 24 25" fill="none">
+                          <path fill-rule="evenodd" clip-rule="evenodd" d="M9.29289 5.43962C9.68342 5.0491 10.3166 5.0491 10.7071 5.43962L16.7071 11.4396C17.0976 11.8301 17.0976 12.4633 16.7071 12.8538L10.7071 18.8538C10.3166 19.2444 9.68342 19.2444 9.29289 18.8538C8.90237 18.4633 8.90237 17.8301 9.29289 17.4396L14.5858 12.1467L9.29289 6.85384C8.90237 6.46331 8.90237 5.83015 9.29289 5.43962Z" fill="#505F79"/>
+                        </svg></li>
+                  </ul>
+                  <hr>
+                  <ul class="work"><li><b>Skills</b></li>
+                      <li><b><svg xmlns="http://www.w3.org/2000/svg" width="24" height="25" viewBox="0 0 24 25" fill="none">
+                          <path fill-rule="evenodd" clip-rule="evenodd" d="M9.29289 5.43962C9.68342 5.0491 10.3166 5.0491 10.7071 5.43962L16.7071 11.4396C17.0976 11.8301 17.0976 12.4633 16.7071 12.8538L10.7071 18.8538C10.3166 19.2444 9.68342 19.2444 9.29289 18.8538C8.90237 18.4633 8.90237 17.8301 9.29289 17.4396L14.5858 12.1467L9.29289 6.85384C8.90237 6.46331 8.90237 5.83015 9.29289 5.43962Z" fill="#505F79"/>
+                        </svg></b></li>
+                  </ul>
+                  <hr>
+              </div>
             </article>
-        </section>
             
-        
-    
-</main> 
+<!-- -----------------  Contact me --------------- -->
+            
+          <article class="self">
+            <img src="asset/images/contactform.png" alt="contact me">
+            <aside class="touch">
+            <h2 class="last">Contact me</h2>
+            <p class="end">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+              I’d love to help with it!</p>
+      
+              <form action="https://formspree.io/f/{form_id}" method="post" >
```

Comment:
- [ ] Good job so far :clap: However, Your form is not submitting successfully, because of the absensence of your `form_id` please resolve it. :slightly_smiling_face: 

![Student form not found](https://github.com/okayboy12/Portfolio-setup-and-mobile-first/assets/43528594/7e6ea2cd-01c4-46f6-b1f3-bdcbaaa225e3)